Code-4-Community/speak-for-the-trees-frontend
# reservation routes in protected api client

## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/925293934/pulses/925415585)

This PR connects the reservation routes to the frontend to allow users to connect to the sftt backend and edit the status of blocks

## This PR

Added a new protected API client file and added each of the routes and functions. Followed the same pattern as the routes made by @jackblanc in LLB.

## Verification Steps

The UI for the reservation pages is not fully set up so I have not been able to confirm that everything is working as expected.
